pennylane/plugins/default_gaussian.py

```diff
@@ -701,10 +701,12 @@ class DefaultGaussian(Device):
 
     Args:
         wires (int): the number of modes to initialize the device in
-        shots (int): How many times should the circuit be evaluated (or sampled) to estimate
-            the expectation values. 0 yields the exact result.
+        shots (int): (default 1) How many times should the circuit be evaluated (or sampled) to estimate
```

Comment:
```suggestion
        shots (int): How many times should the circuit be evaluated (or sampled) to estimate
```
The default value is automatically extracted from the signature by Sphinx, so no need to replicate it here :slightly_smiling_face: